source/agora/common/Types.d

```diff
@@ -76,4 +76,38 @@ public struct Height
 
     /// Provides implicit conversion to `ulong`
     public alias value this;
+
+    /// Helper type for `toString`
+    private alias SinkT = void delegate(scope const(char)[] v) @safe;
+
+    /// Support for Vibe.d serialization to JSON
+    public string toString () const @safe
```

Comment:
Can you also provide a sink overload ?

Reply:
Well, I think we need to make another issue.
A problem with Heat's sink overload has been found.
But it seems to be separate from the related issues.

Reply:
This function leads to new memory allocations when logging, hence why I mentioned it.